Retos/Reto #45 - EL CALENDARIO DE ADEVIENTO 2023 [Fácil]/python/Hool45.py

```diff
@@ -0,0 +1,98 @@
+"""
+¿Conoces el calendario de aDEViento de la comunidad (https://adviento.dev)?
+ * 24 días, 24 regalos sorpresa relacionados con desarrollo de software.
+ * Desde el 1 al 24 de diciembre.
+ *
+ * Crea un programa que simule el mecanismo de participación:
+ * - Mediante la terminal, el programa te preguntará si quieres añadir y borrar
+ *   participantes, mostrarlos, lanzar el sorteo o salir.
+ * - Si seleccionas añadir un participante, podrás escribir su nombre y pulsar enter.
+ * - Si seleccionas añadir un participante, y este ya existe, avisarás de ello.
+ *   (Y no lo duplicarás)
+ * - Si seleccionas mostrar los participantes, se listarán todos.
+ * - Si seleccionas eliminar un participante, podrás escribir su nombre y pulsar enter.
+ *   (Avisando de si lo has eliminado o el nombre no existe)
+ * - Si seleccionas realizar el sorteo, elegirás una persona al azar 
+ *   y se eliminará del listado.
+ * - Si seleccionas salir, el programa finalizará.
+ */
+"""
+
+
+
+
+
+
+
+import random

```

Comment:
maybe its not important but i think that imports should be at top file level

Reply:
thx